extensions/Cake.FtpFolderUpload.yml

```diff
@@ -0,0 +1,16 @@
+Type: Addin
+Name: Cake.FtpFolderUpload
+NuGet: Cake.FtpFolderUpload
+Assemblies:
+- /**/Cake.FtpFolderUpload.dll
+Repository: https://github.com/cake-contrib/cake-ftpfolderupload.git
```

Comment:
Looks like AddinDisco managed to get the right repo this time, following the redirect

Reply:
This is an interesting scenario: the discoverer is still not able to match this repo with any of the Cake-Contrib repos due to the name mismatch (i.e. `Cake.FtpFolderUpload` vs `cake-ftpfolderupload`) but when retrieving the repo from GitHub based on the  repoOwner and repoName (i.e. `jeppevammenkristensen` and `cake-ftpfolderupload`) the metadata returned from the GitHub API reflects the fact that repo has been moved and all URLs point to `https://github.com/cake-contrib/cake-ftpfolderupload`. The discoverer happily uses these updated URLs.